src/screens/VisualTests/VisualTests.tsx

```diff
@@ -266,7 +273,14 @@ export const VisualTests = ({
     testsToStatusUpdate(getFragment(FragmentStatusTestFields, build.testsForStatus.nodes));
 
   useEffect(() => {
-    if (buildStatusUpdate) updateBuildStatus(buildStatusUpdate);
+    if (buildStatusUpdate) {
+      updateBuildStatus((state) => ({
+        ...createEmptyStoryStatusUpdate(state),
+        ...buildStatusUpdate,
+      }));
+    } else {
+      updateBuildStatus(createEmptyStoryStatusUpdate);
+    }
```

Comment:
```suggestion
    updateBuildStatus((state) => ({
      ...createEmptyStoryStatusUpdate(state),
      ...buildStatusUpdate,
    }));
```
This should do the same thing, right? Spreading null/undefined is valid syntax.